<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupValues_0x13">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Активировать</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Состояние</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbState_0x13">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>0x00 – Норма (после сброса)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x07 – ПУСК (идет отсчет времени)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x08 – ОТМЕНА (отсчет времени приостановлен)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x09 – Открытие УРС (УР)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x0A – УРС не открылся</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x0B – УРС открылся</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x0C – ПУСК ПП (выдача пусковых импульсов на ЗПУ)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x0D – Не сработали СДГ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x0E – ПУСК ПРОШЕЛ</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Режим работы</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbRegim_0x13">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>0x00 – Дежурный режим ЗАПРЕТ ПУСКА</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x01 – Дежурный режим АВТОМАТИКА ОТКЛЮЧЕНА</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x03 – Режим пуска  - ПРИОСТАНОВЛЕН ОХТ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x04 – Режим пуска  - МЕСТНЫЙ (РУЧНОЙ)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x05 – Режим пуска  - ДИСТАНЦИОННЫЙ</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0x07 – Режим пуска  - УДАЛЕННЫЙ ОТ СИСТЕМЫ УПРАВЛЕНИЯ</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Состояние УРС</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbURSState_0x13">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>Закрыто</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Частичное открытие</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Открылось</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkOtklVent_0x13">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Флаг необходимости отключения вентиляции</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
